File: Java/ESAMI/Esame24Gennaio2020/Python/Squash.py
from Campo import Campo

class Squash:

    def __init__(self, codice, campo, larghezza, lunghezza, costoOrario, altezza, piano) -> None:
        super().__init__(codice, campo, larghezza, lunghezza, costoOrario)
        self.altezza = altezza
        self.piano = piano

    def __str__(self) -> str:
        return "squash" + "\t" + self.campo + "\t" + str(self.codice) + "\t" + str(self.larghezza) + "\t" + str(self.lunghezza) + "\t" + str(self.altezza) + "\t" + str(self.piano) + "\t" + str(self.costoOrario)

File: SecondoAnno/Java/ESAMI/Esame24Gennaio2020/Java/Squash.java
package Esame24Gennaio2020.Java;

public class Squash extends Campo {

    private float altezza;
    private int piano;

    public Squash(int codice, String campo, float larghezza, float lunghezza, float costoOrario, float altezza, int piano) {
        super(codice, campo, larghezza, lunghezza, costoOrario);
        this.altezza = altezza;
        this.piano = piano;
    }

    @Override
    public String toString() {
        return "squash" + "\t" + campo + "\t" + codice + "\t" + larghezza + "\t" + lunghezza + "\t-\t-\t" + altezza + "\t" + piano + "\t" + costoOrario;
    }
}
